histomicstk/FeatureExtraction.py

```diff
@@ -65,25 +58,23 @@ def FeatureExtraction(Label, I, K=128, Fs=6, Delta=8):
     CytoplasmMeanGradMag	CytoplasmStdGradMag	CytoplasmEntropyGradMag
     CytoplasmEnergyGradMag	CytoplasmSkewnessGradMag	CytoplasmKurtosisGradMag
     CytoplasmSumCanny	CytoplasmMeanCanny
-
     Boundary
     --------
     Boundaries: x1,y1;x2,y2;...
-
-
-
     ##########     Features End     ##########
-
```

Comment:
[Here](https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/histomicstk/SparseColorDeconvolution.py#L51-L57) an example to learn how to add a Reference section